### ISSUE TITLE
Discord link invalid?

### ISSUE THREAD

**Issue by dlasusa** [pola-rs/polars#6215]
### Research

- [X] I have searched the above polars tags on Stack Overflow for similar questions.

- [ ] I have asked my usage related question on [Stack Overflow](https://stackoverflow.com).

### Question about Polars

Apologies if this is the wrong place to ask.

I clicked the discord link from the Polars website.

https://discord.com/invite/4UfP5cfBE7

But it returns as "Invalid" 
![Screenshot 2023-01-13 135406](https://user-images.githubusercontent.com/36902797/212426392-ba92a418-bbb6-4463-8877-791af70f4f75.png)

Just wondering if the issue is on my end or if the link is indeed invalid?

I searched for "polars discord" on stackoverflow, but didn't get any relevant results.

Thanks!

**Comment by alexander-beedie** [comment]
What additional information do you get if you click on the blue _"Why is my invite invalid"_ link?

(I just tried the link and it seemed to work; but now I'm wondering if these links are personalised in some way)

**Comment by dlasusa** [comment]
It's what looks like a standard discord troubleshooting page.  Sorry, I should have included that.

This is where it takes me:
https://support.discord.com/hc/en-us/articles/360001556852

One of the causes can be if I'm a member of 100 discord servers (there is a limit) - but I'm not at that limit.

Thank you for your help.

**Comment by alexander-beedie** [comment]
I thought it might somehow be an expired link on our side, perhaps, but when I click on it there doesn't seem to be an issue. I get a sensible invite page to the channel 🤔

**Comment by dlasusa** [comment]
I tried the 2nd link and it didn't work.  So I then went to the discord app and tried via "add server"  and used both the old and new links.  This time **no error**, it jumped me right into to the Polars discord server.  It makes me think/wonder if I had joined previously?  I can't seem to find a join date, but if that's the case, I'm sorry for wasting your time but very much appreciate the help! And happy to have joined the community!
